src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/config/ResourceServerConfiguration.kt

```diff
@@ -36,8 +36,7 @@ class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
             "/swagger-ui.html",
```

Comment:
not sure why and it's not relevant but I get a 403 for this
https://hmpps-tier-dev.hmpps.service.justice.gov.uk/swagger-ui.html